src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -1393,8 +1393,7 @@
 							</Border.RenderTransform>
 							<FontIcon x:Name="MultiSelectCheck"
 									  FontFamily="{ThemeResource SymbolThemeFontFamily}"
-									  win:Glyph="&#xE73E;"
-									  not_win:Glyph="&#xE081;"
+									  Glyph="&#xE001;"
```

Comment:
You'll need to also change it for GridViewItem (in the same file)

Reply:
Already changed there https://github.com/unoplatform/uno/pull/13349

Reply:
Done!